### DISCUSSION TITLE
[Feature] Add direct install instructions

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [johanohly/AirTrail#4] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi

I generally follow a protocol, to install everything possible inside a LXC (Linuxcontainer) as I run my whole Homelab like many others on Proxmox, support for Docker is not there at all, so I either have to run Docker inside a VM, which often gets into Problems or use LXC.

So I'd appreciate bare-metal installation instruction.

**Comment by johanohly** [comment]
Have you taken a look at the [manual installation method](https://johanohly.github.io/AirTrail/docs/install/manual) in the docs? 

This requires setting up your own postgres database which I should probably mention on that page.

**Reply by CrazyWolf13** [reply]
Oh that's awesome!

I'm so sorry that I missed that!!
